src/frontend/src/common/api/index.tsx

```diff
@@ -20,11 +23,14 @@ export enum ORG_API {
   GET_FASTA_URL = "sequences/getfastaurl",
```

Comment:
should we add a trailing slash here as well to follow the pattern of the other paths?

Reply:
Nope